streams/src/test/java/org/apache/kafka/streams/perf/YahooBenchmark.java

```diff
@@ -305,32 +305,32 @@ public ProjectedEvent apply(ProjectedEvent value) {
             });
 
         // deserialize the add ID and campaign ID from the stored value in Kafka
-        KTable<String, CampaignAd> deserCampaigns = kCampaigns.mapValues(new ValueMapper<String, CampaignAd>() {
+        final KTable<String, CampaignAd> deserCampaigns = kCampaigns.mapValues(new ValueMapper<String, CampaignAd>() {
             @Override
-            public CampaignAd apply(String value) {
-                String[] parts = value.split(":");
-                CampaignAd cAdd = new CampaignAd();
+            public CampaignAd apply(final String value) {
+                final String[] parts = value.split(":");
+                final CampaignAd cAdd = new CampaignAd();
                 cAdd.adID = parts[0];
                 cAdd.campaignID = parts[1];
                 return cAdd;
             }
         });
 
         // join the events with the campaigns
-        KStream<String, String> joined = filteredEvents.join(deserCampaigns,
-            new ValueJoiner<ProjectedEvent, CampaignAd, String>() {
+        final KStream<String, String> joined = filteredEvents.join(deserCampaigns,
+                                                                   new ValueJoiner<ProjectedEvent, CampaignAd, String>() {
```

Comment:
better move `deserCampaigns` to next line to avoid this long indention -- also, if we would keep this large indention, lines below should be indented accordingly, too.